ci/kokoro/macos/build.sh

```diff
@@ -99,11 +76,47 @@ else
   exit 1
 fi
 
+cd "${PROJECT_ROOT}"
+NCPU="$(sysctl -n hw.logicalcpu)"
+readonly NCPU
+
+function google_time() {
+  curl -sI google.com | grep "^Date:" | cut -f2- -d: | tr -d '\r' | xargs
```

Comment:
This seems overly convoluted, down to the `xargs` whose purpose seems to be to remove the leading space.

How about just ...
```
  curl -sI google.com | sed -n 's/Date: \(.*\)\r/\1/p'
```

Also, should we update the `google_time()` in `ci/cloudbuild/build.sh` to also be wary of the CR?

Reply:
done